2020/day-1-report-repair/index.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { findSumParts } from './'
```

Comment:
For my own education, how does this work without passing in a module name? Does it use `index` by default?

Reply:
Yep! Could also do `'./index`